wger/utils/fields.py

```diff
@@ -45,5 +46,6 @@ def formfield(self, **kwargs):
         Use our custom field
         '''
         defaults = {'form_class': Html5FormDateField}
+
         defaults.update(kwargs)
         return super(Html5DateField, self).formfield(**defaults)
```

Comment:
these two fields are similar, maybe you should look at them

Reply:
Thanx @peterwade153 , Peter but one is date and the other is time.